src/lib/drivers/bm_dfu_wrapper.cpp

```diff
@@ -0,0 +1,93 @@
+#include "bm_dfu_generic.h"
+#include "bootutil/bootutil_public.h"
+#include "configuration.h"
+#include "flash_map_backend/flash_map_backend.h"
+#include "nvmPartition.h"
+#include "reset_reason.h"
+#include "stm32_flash.h"
+#include "sysflash/sysflash.h"
```

Comment:
implement all of the DFU functions that require sys config access, flash/NVM partition read, writes, open, closes, and all MCUBoot/resetting functions.